src/components/SVGViewer/SVGViewerSearch.tsx

```diff
@@ -211,7 +211,7 @@ class SVGViewerSearch extends React.Component<ComponentProps, ComponentState> {
     currentSearchResult.classList.remove(currentSearchClassName);
     searchResults[nextResult - 1].classList.add(currentSearchClassName);
     this.setState({ currentResult: nextResult });
-    zoomOnCurrentAsset(searchResults[nextResult - 1]);
+    onSearchResult(searchResults[nextResult - 1]);
```

Comment:
Perhaps it's a good idea to only call this method if `searchResults[nextResult - 1]` exists? That'd allow us to not having to check for `null` values in `SVGViewer#handleSearchResult`

Reply:
makes sense actually